tests/test_es_models.py

```diff
@@ -23,7 +26,10 @@ def test_missing_required_args(input_args):
 def test_get_es_models_standard_behavior():
     models = get_es_models()
 
-    assert len(models) == 11
+    models_dir = pkg_resources.resource_filename('gdcmodels', 'es-models')
```

Comment:
This is really close to the code being tested. Maybe the hardcoded number is better in some way.

Reply:
yeah, makes sense.